docker/package/tezos_baking_wizard.py

```diff
@@ -210,166 +245,121 @@ def pprint_options(self):
-    )
-    print(
-        "In order to run a baking instance, you'll need the following Tezos binaries:\n",
-        "tezos-client, tezos-node, tezos-baker-<proto>, tezos-endorser-<proto>.\n",
-        "If you have installed tezos-baking, these binaries are already installed.",
-    )
-    print(
-        "To access help and possible options for each question, type in 'help' or '?'."
-    )
-    print("Type in 'exit' to quit.")
-    print()
+# Steps
+
+network_query = Step(
+    id="network",
+    prompt="Which Tezos network would you like to use?\nCurrently supported:",
+    help="The selected network will be used to set up all required services.\n"
+    "The currently supported protocols are 009-PsFLoren (used on florencenet and mainnet)\n"
+    "and 010-PtGRANAD (granadanet).\nKeep in mind that you must select the test network "
+    "(either florencenet or granadanet) if you plan on baking with a faucet JSON file.",
+    options=networks,
+    validator=Validator(enum_range_validator(networks)),
+)
+
+service_mode_query = Step(
+    id="mode",
+    prompt="Do you want to set up baking or to run the standalone node?",
+    help="By default, tezos-baking provides predefined services for running baking instances "
+    "on different networks.\nSometimes, however, you might want to only run the Tezos node.\n"
+    "When this option is chosen, this wizard will help you bootstrap tezos-node only.",
+    options=modes,
+    validator=Validator(enum_range_validator(modes)),
+)
+
+systemd_mode_query = Step(
+    id="systemd_mode",
+    prompt="Would you like your setup to automatically start on boot?",
+    help="Starting the service will make it available just for this session, great\n"
+    "if you want to experiment. Enabling it will make it start on every boot.",
+    options=systemd_enable,
+    validator=Validator(enum_range_validator(systemd_enable)),
+)
+
+snapshot_mode_query = Step(
+    id="snapshot",
+    prompt="The Tezos node can take a significant time to bootstrap from scratch.\n"
+    "Bootstrapping from a snapshot is suggested instead.\n"
+    "How would you like to proceed?",
+    help="A fully-synced local tezos-node is required for running a baking instance.\n"
+    "By default, service with tezos-node will start to bootstrap from scratch,\n"
+    "which will take a significant amount of time.\nIn order to avoid this, we suggest "
+    "bootstrapping from a snapshot instead.\n\n"
+    "Snapshots can be downloaded from the following websites:\n"
+    "Tezos Giganode Snapshots - https://snapshots-tezos.giganode.io/ \n"
+    "XTZ-Shots - https://xtz-shots.io/ \n\n"
+    "We recommend to use rolling snapshots. This is the smallest and the fastest mode\n"
+    "that is sufficient for baking (you can read more about other tezos-node history modes here:\n"
+    "https://tezos.gitlab.io/user/history_modes.html#history-modes ).\n"
+    "All commands within the service are run under the 'tezos' user.",
```

Comment:
Not sure if this last sentence is actually related to this step:)
Perhaps, it can be moved to the welcome message